dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,79 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+You can increase the size of the persistent volumes (PVs) that your pods are using. To do this, you can update the persistent volume claim (PVC) to request a new size. The  controller then applies the change to the PV, which is bound to the PVC.
+
+For volume types that require both file system expansion and a volume plugin-based modification, growing PVs is a two-step process. For volume types that only require a volume plugin-based API call, growing PVs is a one-step process.
+
+A new controller called `volume_expand_controller` listens for PVC size expansion requests and takes action as needed. The new controller does the following:
```

Comment:
It appears that we are documenting internals of Volume Expand Controller here. There is no harm in doing that but the top section of this documentation should start with "How to Expand PVCs". Not with how volume expansion works.

Reply:
We shouldn't have this, only the UX.. it will confuse users.